hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -121,6 +121,10 @@
   private static final String MERGE_DATA_VALIDATION_CHECK_ENABLED = "hoodie.merge.data.validation.enabled";
   private static final String DEFAULT_MERGE_DATA_VALIDATION_CHECK_ENABLED = "false";
 
+  // payload ordering field
+  private static final String PAYLOAD_ORDERING_FIELD_PROP = "hoodie.payload.ordering.field";
```

Comment:
Is there any reason we need a different config from the pre-combine field? IIUC we wanna make the merging logic consistent, so `preCombine` and `combineAndGetUpdateValue` should produce the same result?